tutorials/images/Images_Tutorial.ipynb

```diff
@@ -0,0 +1,444 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%load_ext autoreload\n",
+    "%autoreload 2\n",
+    "\n",
+    "import numpy as np\n",
+    "import matplotlib.pyplot as plt\n",
+    "%matplotlib inline\n",
+    "\n",
+    "import os"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "If you don't already have `skimage` installed, please run the following cell."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import sys\n",
+    "!{sys.executable} -m pip install scikit-image"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Load and Visualize Dataset\n",
+    "First, we load the dataset and associated bounding box objects and labels.\n",
+    "Note that in some cases, the resulting images from the query will be incorrectly labeled. THis is because the ground truth also comes from a crodsourced setting where relations in the image are described, and therefore prone to error."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from data_loader import DataLoader\n",
+    "loader = DataLoader()\n",
+    "\n",
+    "loader.show_examples(annotated=False, label=1)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Generate Primitives\n",
+    "Since we cannot check relations in the images directly by writing labeling functions over the primitives, we first extract a set of  \"primitives\" from the images that are easily interpretable.\n",
+    "\n",
+    "In this case, we use information about the objects in the data such as their labels, their positions and their size.  We show how we develop simple primitives that are based on the labels of the different objects and include the more complex primitives in `primitive_helpers.py`."
```

Comment:
Also: make it clear that these primitives can be generated by generic off the shelf tools that are more broadly applicable then our specific problem- and that they can be noisy!

Reply:
+1. also would be nice to mention we define a limited set of primitives here but technically we could use other information about objects in the image etc.